openproblems/tasks/cell_cell_communication/metrics/odds_ratio.py

```diff
@@ -4,7 +4,7 @@
 import scipy.stats as stats
 
 
-@metric(metric_name="Odds Ratio", maximize=True)
+@metric(metric_name="Odds Ratio", maximize=True, image="openproblems-r-extras")
```

Comment:
I don't think this needs `r-extras`. The `lr_resource` error should be fixed by my latest commit